### PR TITLE
A small revision of the jobflow tutorial

### PR DESCRIPTION
**Pull request by QuantumChemist**

### CODE REVIEW

File: jobflow.ipynb [hunk 1]
```diff
@@ -908,11 +914,20 @@
    "cell_type": "markdown",
    "source": [
     "## Publication of the workflow\n",
-    "The `jobflow` infrastructure does not provide a dedicated platform for publishing a workflow currently. However, workflows related to computational materials science have been collected in the package `atomate2`. In addition, users can build their own package by relying on jobflow and share it as a new Python-based program. Additional packages in materials science using `jobflow` exist."
+    "The `jobflow` infrastructure does not provide a dedicated platform for publishing a workflow currently. However, workflows related to computational materials science have been collected in the package `atomate2`. In addition, users can build their own package by relying on `jobflow` and share it as a new Python-based program. There are also additional materials science packages that rely on `jobflow`."
```
Comment (QuantumChemist) [inline jobflow.ipynb]: @JaGeo , do you know an example outside of the MP world?

Comment (JaGeo) [reply jobflow.ipynb]: Check out the publication on jobflow plus QuAcc.

Comment (QuantumChemist) [reply jobflow.ipynb]: Although QuAcc feels like half part of the MP 😃

**Comment by QuantumChemist** [comment]
I am pretty satisfied with this version. If you want, you can merge it @JaGeo :)

**Comment by JaGeo** [comment]
I am happy with it as well. I will merge it into the main branch!

**Pull request merged**